Avuidrauxs/interactive-timer-project-001
# Feature: The message above the timer should reflect how much time has passed

#### What does this PR do?
It makes the message displayed above the timer depend on what part of a time inputted by user has already passed. The way it is accomplished is by setting `initialTime` stateful value to equal `input` provided by the user when they click START button and therefore run `startTimer()` function. `initialTime` is not changed when calling `resumeTimer()`. `ClockComment.js` is rerendered each time `timeLeft` value is changed and therefore `progress` value is recalculated.
#### Description of Task to be completed?
Add a feature making the message above the timer reflect how much time has passed
#### How should this be manually tested?
Run timer at different speeds and values and check whether the message changes at each breakpoint (0% has passed -> 25% has passed -> 50% -> 75% -> 100%)
